2024-02-29 16:50:43,313 INFO    MainThread:16296 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-29 16:50:43,314 INFO    MainThread:16296 [wandb_setup.py:_flush():76] Configure stats pid to 16296
2024-02-29 16:50:43,314 INFO    MainThread:16296 [wandb_setup.py:_flush():76] Loading settings from /home/mlops/.config/wandb/settings
2024-02-29 16:50:43,314 INFO    MainThread:16296 [wandb_setup.py:_flush():76] Loading settings from /home/mlops/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/settings
2024-02-29 16:50:43,314 INFO    MainThread:16296 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-29 16:50:43,314 INFO    MainThread:16296 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-29 16:50:43,314 INFO    MainThread:16296 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-29 16:50:43,315 INFO    MainThread:16296 [wandb_init.py:_log_setup():526] Logging user logs to /home/mlops/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240229_165043-1g8yxlad/logs/debug.log
2024-02-29 16:50:43,315 INFO    MainThread:16296 [wandb_init.py:_log_setup():527] Logging internal logs to /home/mlops/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240229_165043-1g8yxlad/logs/debug-internal.log
2024-02-29 16:50:43,315 INFO    MainThread:16296 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7efe001e6a00>
2024-02-29 16:50:43,316 INFO    MainThread:16296 [wandb_init.py:init():566] calling init triggers
2024-02-29 16:50:43,316 INFO    MainThread:16296 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-02-29 16:50:43,316 INFO    MainThread:16296 [wandb_init.py:init():616] starting backend
2024-02-29 16:50:43,316 INFO    MainThread:16296 [wandb_init.py:init():620] setting up manager
2024-02-29 16:50:43,322 INFO    MainThread:16296 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-29 16:50:43,325 INFO    MainThread:16296 [wandb_init.py:init():628] backend started and connected
2024-02-29 16:50:43,333 INFO    MainThread:16296 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-29 16:50:43,334 INFO    MainThread:16296 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-29 16:50:43,334 INFO    MainThread:16296 [wandb_init.py:init():720] updated telemetry
2024-02-29 16:50:43,358 INFO    MainThread:16296 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-29 16:50:46,438 INFO    MainThread:16296 [wandb_run.py:_on_init():2262] communicating current version
2024-02-29 16:50:46,581 INFO    MainThread:16296 [wandb_run.py:_on_init():2271] got version response 
2024-02-29 16:50:46,581 INFO    MainThread:16296 [wandb_init.py:init():804] starting run threads in backend
2024-02-29 16:50:49,870 INFO    MainThread:16296 [wandb_run.py:_console_start():2241] atexit reg
2024-02-29 16:50:49,870 INFO    MainThread:16296 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-29 16:50:49,870 INFO    MainThread:16296 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-29 16:50:49,871 INFO    MainThread:16296 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-29 16:50:49,872 INFO    MainThread:16296 [wandb_init.py:init():847] run started, returning control to user process
2024-02-29 16:50:51,249 INFO    MainThread:16296 [jupyter.py:_save_ipynb():384] looking for notebook: src/eda/eda.ipynb
2024-02-29 16:50:51,250 INFO    MainThread:16296 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-29 16:50:51,256 INFO    MainThread:16296 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-29 16:50:51,274 INFO    MainThread:16296 [jupyter.py:_save_ipynb():384] looking for notebook: src/eda/eda.ipynb
2024-02-29 16:50:51,274 INFO    MainThread:16296 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-29 16:50:51,281 INFO    MainThread:16296 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-29 16:50:51,328 INFO    MainThread:16296 [jupyter.py:_save_ipynb():384] looking for notebook: src/eda/eda.ipynb
2024-02-29 16:50:51,329 INFO    MainThread:16296 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-29 16:50:51,334 INFO    MainThread:16296 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-29 16:50:51,349 INFO    MainThread:16296 [jupyter.py:_save_ipynb():384] looking for notebook: src/eda/eda.ipynb
2024-02-29 16:50:51,349 INFO    MainThread:16296 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-29 16:50:51,355 INFO    MainThread:16296 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-29 16:50:51,376 INFO    MainThread:16296 [jupyter.py:_save_ipynb():384] looking for notebook: src/eda/eda.ipynb
2024-02-29 16:50:51,376 INFO    MainThread:16296 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-29 16:50:51,381 INFO    MainThread:16296 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-29 16:50:51,403 INFO    MainThread:16296 [jupyter.py:_save_ipynb():384] looking for notebook: src/eda/eda.ipynb
2024-02-29 16:50:51,403 INFO    MainThread:16296 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-29 16:50:51,409 INFO    MainThread:16296 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-29 16:50:51,410 INFO    MainThread:16296 [wandb_run.py:_finish():1970] finishing run afowl78/nyc_airbnb/1g8yxlad
2024-02-29 16:50:51,464 INFO    MainThread:16296 [jupyter.py:save_history():474] saving 8 cells to _session_history.ipynb
2024-02-29 16:50:51,464 INFO    MainThread:16296 [wandb_run.py:_config_callback():1343] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2024-02-29 16:50:51,468 INFO    MainThread:16296 [jupyter.py:_save_ipynb():384] looking for notebook: src/eda/eda.ipynb
2024-02-29 16:50:51,469 INFO    MainThread:16296 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-29 16:50:51,469 INFO    MainThread:16296 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-29 16:50:51,469 INFO    MainThread:16296 [wandb_run.py:_restore():2193] restore
2024-02-29 16:50:51,469 INFO    MainThread:16296 [wandb_run.py:_restore():2199] restore done
2024-02-29 16:50:57,424 INFO    MainThread:16296 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-29 16:50:57,424 INFO    MainThread:16296 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-29 16:50:57,439 INFO    MainThread:16296 [wandb_run.py:_footer_sync_info():3825] logging synced files
